retrievalmarket/types/transports.ipldsch

```diff
@@ -3,7 +3,8 @@
 type Multiaddr bytes
 
 type Protocol struct {
-  # The name of the transport protocol eg "libp2p" or "http"
+  # The name of the transport protocol
+  # Known protocols: "libp2p", "http", "https"
```

Comment:
@dirkmc @hannahhoward where does `libp2p-http` fit in this scenario?

Reply:
Currently `libp2p-http` is only used for storage deals